charts/ontology/templates/service.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
```

```suggestion
  labels:
    app: {{ .Chart.Name }}-service
```
If you want to include a label to indicate that this is the ontology service. It should be added to the labels on the service itself rather than be in the seletor